$OpenBSD: patch-tests_playTests_sh,v 1.10 2018/06/29 19:02:07 bket Exp $

Revert https://github.com/facebook/zstd/pull/1124 as it causes 'make test' to
fail. This is explained by a test trying to write to /dev/random, which fails
as a regular user is not allowed to write to this device. /dev/zero has
different permissions set.

Index: tests/playTests.sh
--- tests/playTests.sh.orig
+++ tests/playTests.sh
@@ -56,7 +56,7 @@ fi
 
 isWindows=false
 INTOVOID="/dev/null"
-DEVDEVICE="/dev/random"
+DEVDEVICE="/dev/zero"
 case "$OS" in
   Windows*)
     isWindows=true
